platform/darwin/src/NSValue+MGLStyleEnumAttributeAdditions.h

```diff
@@ -0,0 +1,150 @@
+// This file is generated.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import <Foundation/Foundation.h>
+
+#import "MGLFillStyleLayer.h"
+#import "MGLLineStyleLayer.h"
+#import "MGLSymbolStyleLayer.h"
+#import "MGLCircleStyleLayer.h"
+#import "MGLRasterStyleLayer.h"
+#import "MGLBackgroundStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSValue (MGLStyleEnumAttributeAdditions)
```

Comment:
Actually I only see (even in our current release version) one "other" category https://www.mapbox.com/ios-sdk/api/3.3.7/Other%20Categories.html